am-lib/src/main/java/uk/gov/hmcts/reform/amlib/DefaultRoleSetupImportService.java

```diff
@@ -0,0 +1,128 @@
+package uk.gov.hmcts.reform.amlib;
+
+import lombok.NonNull;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.sqlobject.SqlObjectPlugin;
+import uk.gov.hmcts.reform.amlib.enums.AccessManagementType;
+import uk.gov.hmcts.reform.amlib.enums.RoleType;
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+import uk.gov.hmcts.reform.amlib.exceptions.ErrorAddingEntriesToDatabaseException;
+import uk.gov.hmcts.reform.amlib.models.DefaultPermission;
+import uk.gov.hmcts.reform.amlib.models.DefaultPermissionGrant;
+import uk.gov.hmcts.reform.amlib.repositories.DefaultRoleSetupRepository;
+import uk.gov.hmcts.reform.amlib.utils.Permissions;
+
+public class DefaultRoleSetupImportService {
+    private final Jdbi jdbi;
+
+    public DefaultRoleSetupImportService(String url, String user, String password) {
+        this.jdbi = Jdbi.create(url, user, password);
+        this.jdbi.installPlugin(new SqlObjectPlugin());
+    }
+
+    public void addService(@NonNull String serviceName, String serviceDescription) {
+        if (serviceName.isEmpty()) {
+            throw new IllegalArgumentException("Service name cannot be empty");
+        }
+
+        jdbi.useExtension(DefaultRoleSetupRepository.class,
+            dao -> dao.addService(serviceName, serviceDescription));
+    }
+
+    public void addRole(@NonNull String roleName, RoleType roleType, SecurityClassification securityClassification,
```

Comment:
Can role type and security classification be null? They seem to be NOT_NULL at DB side.

Reply:
you're right, I thought that a type check on RoleType wouldn't allow a null value through. Shall add lombok annotations